### DISCUSSION TITLE
Modals routing / Parallel routing

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#24058] [Questions]
Hey guys

Is this possible to do something similar in nuxt by more native approach? 
https://github.com/theguriev/modals-concept

I'm trying to connect routing and modals somehow. In NextJS it's possible to do by using App Router + Parallel Route https://nextjs.org/docs/app/building-your-application/routing/parallel-routes 
Do we have something similar in Nuxt? @danielroe @pi0

**Comment by theguriev** [comment]
Here is the link to the deployed project https://modals-concept.vercel.app/

**Comment by serkodev** [comment]
@theguriev If you still need it, welcome to check out the module I developed to implement Parallel Routes / Modal Routes in Nuxt.

https://nuxt-pages-plus.pages.dev/
